test/integration/discovery.v1.test.js

```diff
@@ -105,12 +104,10 @@ describe('discovery v1 integration', () => {
   it('should perform a federated query for notices', async () => {
     const params = {
       environmentId,
-      configurationId: configurationId,
       collectionIds: [collectionId, collectionId2],
       filter: 'yesplease',
       count: 10,
       sort: ['+field_1', '-field_2'],
-      natural_language_query: 'a question about stuff and things',
```

Comment:
why is `natural_language_query` removed?

Reply:
This is no longer a valid parameter